src/cheri_insts.sail

```diff
@@ -89,20 +126,33 @@ function clause execute (CGetLen(rd, cs1)) =
   RETIRE_SUCCESS
 }
 
+/*!
+ * The low bit of integer register *rd* is set to the **tag** value of *cs1*.
+ * All other bits of *rd* are cleared.
+ */
 function clause execute (CGetTag(rd, cs1)) =
 {
   let capVal = C(cs1);
   X(rd) = EXTZ(bool_to_bits(capVal.tag));
   RETIRE_SUCCESS
 }
 
+/*!
+ * The low bit of integer register \emph{rd} is set to 0 if *cs1* is unsealed
+ * and to 1 otherwise.
+ * All other bits of *rd* are cleared.
+ */
 function clause execute (CGetSealed(rd, cs1)) =
 {
   let capVal = C(cs1);
   X(rd) = EXTZ(bool_to_bits(capVal.sealed));
   RETIRE_SUCCESS
 }
 
+/*!
+ * Integer register *rd* is set equal to the **cursor** field (the address) of
```

Comment:
The field is called address in RISC-V.

Reply:
We still use `getCapCursor`, so until that is changed we should maybe match the MIPS description?

Reply:
Those 3 instances should just be replaced with `cs2.address` like everywhere else and the function removed.